docs/src/main/asciidoc/spanner.adoc

```diff
@@ -25,51 +27,180 @@ dependencies {
 }
 ----
 
+This setup takes care of bringing in the latest compatible version of Cloud Java Spanner libraries as well.
+
+== Spring Data Spanner
+
+
+=== Configuration
+
+To setup Spring Data Spanner, you have to configure connection to Cloud Spanner and Repositories (if you want to use them).
```

Comment:
you have to configure **a** connection to Cloud Spanner. If you want to use **Cloud Repositories**, you must set up a new connection for them, too. (?)